polynote-kernel/src/test/scala/polynote/kernel/interpreter/python/PythonInterpreterSpec.scala

```diff
@@ -268,6 +268,18 @@ class PythonInterpreterSpec extends FreeSpec with Matchers with InterpreterSpec
           |  def _repr_latex_(self):
           |    return "latex{string}"
           |
+          |  def _repr_svg_(self):
+          |    return "<svg />"
+          |
+          |  def _repr_jpeg_(self):
+          |    return ("somekindofbase64encodedjpeg", {'height': 400 })
```

Comment:
Note the tuple here, see https://ipython.readthedocs.io/en/stable/config/integrating.html#rich-display

> We often want to provide frontends with guidance on how to display the data. To support this, `_repr_*_()` methods can also return a `(data, metadata)` tuple where metadata is a dictionary containing arbitrary key-value pairs for the frontend to interpret. An example use case is `_repr_jpeg_()`, which can be set to return a jpeg image and a `{'height': 400, 'width': 600}` dictionary to inform the frontend how to size the image.

In the current implementation the metadata is not read.